### PR TITLE
Add support for running on specific version of EdgeHub image

### PR DESCRIPTION
**Pull request by marianan**
Add support for running on specific version of EdgeHub image (#105)

Validation: ran unit tests, added 2 new end to end tests.

Updated version to create a new release if CICD finds no issues.

### CODE REVIEW

**Review by adashen — COMMENTED** [review]
Seems failed the travis build because of some code style issue.

**Comment by marianan** [comment]
Thanks for noting this, all issues are now resolved.

---
In reply to: [498162841](https://github.com/Azure/iotedgehubdev/pull/284#issuecomment-498162841) [](ancestors = 498162841)

File: tests/test_cli.py [hunk 1]
```diff
@@ -501,6 +500,37 @@ def test_cli_start_with_create_options_for_bind(runner):
...
+            assert 'IoT Edge Simulator has been started in single module mode' in output
+            assert 'curl --header' in output
+        else:
+            raise Exception(result.stdout)
+    finally:
+        result = cli_stop(runner)
+        assert 'IoT Edge Simulator has been stopped successfully' in result.output.strip()
+        remove_docker_networks(['azure-iot-edge-dev'])
+        remove_docker_images(['mcr.microsoft.com/azureiotedge-simulated-temperature-sensor:1.0',
+                              'mcr.microsoft.com/azureiotedge-hub:1.0.9.5',
```
Comment (adashen) [inline tests/test_cli.py:519]: 1.0.9.5 or 1.0.9.4?

File: tests/test_cli.py [hunk 2]
```diff
@@ -119,7 +119,6 @@ def cli_start_with_deployment(runner, deployment_json_file_path):
     if 'IoT Edge Simulator has been started in solution mode' not in result.output.strip():
         raise Exception(result.stdout)
 
-
 def invoke_module_method():
```
Comment (mamokarz) [inline tests/test_cli.py:122]: Consider keep this space.

**Review by mamokarz — APPROVED** [review]
:shipit:

**Pull request merged**